examples/react/basic/src/main.tsx

```diff
@@ -172,3 +173,8 @@ if (!rootElement.innerHTML) {
 
   root.render(<RouterProvider router={router} />)
 }
+
+// Test that `Link` can be supplied its properties from a destructed `LinkOptions` object without causing a type error.
+// TODO: Move into a defined test once a testing framework is setup.
+const testLinkOptions: LinkOptions = { to: '/' }
```

Comment:
Maybe change this to use LinkProps? or possibly add another option with that type?

Since the LinkProps type is what is meant to be the base for what is accepted by the <Link> component.

Reply:
Makes sense, update to `LinkProps` :)